packages/beacon-node/src/eth1/index.ts

```diff
@@ -70,7 +70,7 @@ export class Eth1ForBlockProduction implements IEth1ForBlockProduction {
 
     this.eth1DepositDataTracker = opts.disableEth1DepositDataTracker
       ? null
-      : new Eth1DepositDataTracker(opts, modules, eth1Provider);
+      : new Eth1DepositDataTracker({...opts, enabled: false}, modules, eth1Provider);
```

Comment:
just disabled this as of now, requires more code changes to correctly stop deposit data tracker